core/session.py

```diff
@@ -49,41 +46,43 @@ class GaeSession:
 
     """
     kindName = "viur-session"
-    sameSite = "lax"  # Either None (dont issue sameSite header), "none", "lax" or "strict"
+    sameSite = "lax"  # Either None (don't issue sameSite header), "none", "lax" or "strict"
     sessionCookie = True  # If True, issue the cookie without a lifeTime (will disappear on browser close)
-    cookieName = f'viurCookie_{conf["viur.instance.project_id"]}'
-
-    def load(self, req: BrowseHandler):
-        """
-            Initializes the Session.
+    cookieName = f'viur_cookie_{conf["viur.instance.project_id"]}'
 
-            If the client supplied a valid Cookie, the session is read
-            from the datastore, otherwise a new, empty session
-            will be initialized.
-        """
+    def __init__(self):
+        super().__init__()
         self.changed = False
         self.isInitial = False
         self.cookieKey = None
         self.sslKey = None
         self.staticSecurityKey = None
         self.securityKey = None
         self.session = {}
+
+    def load(self, req: BrowseHandler):
+        """
+            Initializes the Session.
+
+            If the client supplied a valid Cookie, the session is read from the datastore, otherwise a new,
+            empty session will be initialized.
+        """
         if self.cookieName in req.request.cookies:
             cookie = str(req.request.cookies[self.cookieName])
-            data = db.Get(db.Key(self.kindName, cookie))
-            if data:  # Loaded successfully from Memcache
-                if data["lastseen"] < time() - conf["viur.session.lifeTime"]:
+            if data := db.Get(db.Key(self.kindName, cookie)):  # Loaded successfully from Memcache
```

Comment:
There's no longer a memcache behind 
```suggestion
            if data := db.Get(db.Key(self.kindName, cookie)):  # Loaded successfully
```